quantum/process_keycode/process_ucis.c

```diff
@@ -93,6 +93,10 @@ void register_ucis(const char *hex) {
 }
 
 bool process_ucis (uint16_t keycode, keyrecord_t *record) {
+  uint8_t i;
+
+  unicode_input_mode_init();
```

Comment:
`unicode_input_mode_init` shouldn't be called here.

Reply:
Yup, poor merging on my part